### ISSUE TITLE
Working Classifier

### ISSUE THREAD

**Issue by MJC598** [MJC598/RJI-Software-Engineering-Project#3]
Description: Currently we do not have a technical classifier, nor the dependencies for it, functioning. The biggest and most painful issues being the dependency libsvm for the brisque module. It will not install by any means, and we have not been able to discern why or fix any of the errors given during install with pip. Also running into issues with matplotlib install with pip, as we cannot install it as user nor root, as attempting to with either gives us an error that tells us to install it as the other (root -> user or user -> root).

Steps to reproduce:
1. Open a terminal and connect to the virtual environment we have setup on the ec2 instance
2. Enter in the command `pip install matplotlib --user` or `sudo pip install matplotlib`
3. The errors in red are the problems we are facing currently

Expected behavior: We expect the dependencies to  install with either `sudo pip install matplotlib` or `pip install matplotlib --user` but neither works. These issues can extend in similar fashion to the other dependencies such as  libsvm (installed with package manager or by rpm file).

Desktop:
OS: Amazon Linux AMI
Browser: All browsers

Additional context:

**Comment by Deimious** [comment]
We have changed servers from an Amazon Linux AMI one to an Ubuntu one so we could take advantage of the easier-to-use and more-up-to-date apt package manager. We have also changed the dependencies as it was just not working with libsvm or brisque. We've moved to idealo with a TensorFlow backend. Currently are dependency issues are fixed and the classifier is working, albeit on a single picture at a time. I'll be creating a new issue for this as this one has been fixed.
